modules/using-assisted-installer-oci-agent-iso.adoc

```diff
@@ -4,16 +4,21 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="using-assisted-installer-oci-agent-iso_{context}"]
-= Using the Assisted Installer to generate an OCI-compatible discovery ISO image
+= Using the {ai-full} to generate an {oci}-compatible discovery ISO image
 
 Generate a discovery ISO image and upload the image to {oci-first}, so that the agent can perform hardware and network validation checks before you install an {product-title} cluster on {oci}.
 
+From the {oci} web console, you must create the following resources:
+
+* A compartment for better organizing, restricting access, and setting usage limits to {oci} resources.
+* An object storage bucket for safely and securely storing the discovery ISO image. You can access the image at a later stage for the purposes of booting the instances, so that you can then create your cluster.
+
 .Prerequisites
 
-* You created a child compartment and an object storage bucket on {oci}. See the "Creating OCI resources and services" section.
+* You created a child compartment and an object storage bucket on {oci}. See link:https://docs.oracle.com/iaas/Content/openshift-on-oci/installing-assisted.htm#provisioning-cloud-infrastructure[Provisioning Cloud Infrastructure ({oci} Console)] in the _Oracle_ documentation.
 * You reviewed details about the {product-title} installation and update processes.
 * If you use a firewall and you plan to use a Telemetry service, you configured your firewall to allow {product-title} to access the sites required.
-* Before you create a virtual machines (VM), refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
+* Before you create a virtual machines (VM), reference {op-system-base-full} certified shapes and instance types to identify the supported {oci} VM shapes. For more information about supported Oracle virtual machines, see link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types (Red Hat Ecosystem Catalog portal)].
```

Comment:
I know zero about Oracle shapes and images, but this sentence still sound awkward to me. Is this way off? 
```suggestion
* Before you create a virtual machines (VM), see link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types (Red Hat Ecosystem Catalog portal)] to identify the supported {oci} VM shapes.
```